client-src/index.js

```diff
@@ -21,9 +21,8 @@ const defaultOptions = {
   hotReload: true,
   liveReload: false,
   initial: true,
-  useWarningOverlay: false,
-  useErrorOverlay: false,
-  useProgress: false,
+  progress: false,
+  overlay: false,
```

Comment:
Better keep options here as is,in future we can improve this and allow to using this `http://192.168.0.5?overlay&no-progress`